tests/setup.ts

```diff
@@ -1,18 +1,21 @@
-import { GenericContainer, StartedTestContainer, TestContainer, Wait } from 'testcontainers';
+import type { StartedTestContainer } from 'testcontainers';
+import { Wait, PostgreSqlContainer } from 'testcontainers';
 
-let startedContainer: StartedTestContainer;
+const containers: StartedTestContainer[] = [];
 
-export const setup = async () => {
-    const container: TestContainer = new GenericContainer('postgres');
-    startedContainer = await container
-        .withEnvironment({ POSTGRES_USER: 'postgres' })
-        .withEnvironment({ POSTGRES_PASSWORD: 'nango_test' })
-        .withEnvironment({ POSTGRES_DB: 'postgres' })
+async function setupPostgres() {
+    const container = new PostgreSqlContainer('postgres:15.5-alpine');
+    const pg = await container
+        .withDatabase('postgres')
```

Comment:
variable for this since it is repeated with `NANGO_DB_NAME`

Reply:
good point 👌🏻